python/ray/dashboard/dashboard.py

```diff
@@ -946,7 +946,7 @@ def clean_trials(self, trial_details):
         default=None,
         help="Specify the path of the temporary directory use by Ray process.")
     args = parser.parse_args()
-    ray.utils.setup_logger(args.logging_level, args.logging_format)
+    ray.ray_logging.setup_logger(args.logging_level, args.logging_format)
```

Comment:
Should this just be `ray.logging` instead of `ray.ray_logging`? Seems redundant.

Reply:
Actually, I tried with logging initially, and it had some weird failures (I guess because of some module name conflicts). For example, this won't work

```python3
# logging.py
import logging
logging.basicConfig(level=logging.INFO)
```

Reply:
I can technically fix it by importing like

```python3
# logging.py
import logging as ray_logging
```
I think. So let me know if you'd prefer this approach.